src/System.Net.Primitives/tests/FunctionalTests/CredentialCacheTest.cs

```diff
@@ -94,6 +98,37 @@ private static IEnumerable<CredentialCacheCount> GetCredentialCacheCounts()
             yield return CreateHostPortCredentialCacheCount();
             yield return CreateUriAndHostPortCredentialCacheCount();
         }
+        
+        public static IEnumerable<object[]> StandardAuthTypeWithNetworkCredential =>
+            new[]
+            {
+                new object[] {authenticationTypeNTLM, credential1},
+                new object[] {authenticationTypeKerberos, credential2},
+                new object[] {authenticationTypeNegotiate, credential3},
+                new object[] {authenticationTypeBasic, credential4},
+                
+                new object[] {authenticationTypeNTLM, CredentialCache.DefaultNetworkCredentials as NetworkCredential},
+                new object[] {authenticationTypeKerberos, CredentialCache.DefaultNetworkCredentials as NetworkCredential},
+                new object[] {authenticationTypeNegotiate, CredentialCache.DefaultNetworkCredentials as NetworkCredential},
+                // Default credentials cannot be supplied for the Basic authentication scheme.
```

Comment:
>Instead of a comment, please capture this in a test.

I'm not sure how to do this, should I: make a new test case for this particular test data, or use a if statement in my previous test function? Because this is a special case, and my test function name is Add_UriAuthenticationType_Success(). Adding if statement in the function seems not a good choice.

But this particular test data belongs to StandardAuthTypeWithNetworkCredential{} group, should I make a new collection EdgeCases{} which only contains these two combination:
- default credential with Basic scheme
- default credential with Digest scheme